lib/content-services/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -26,66 +26,59 @@ import { CardViewGroup } from '../../interfaces/content-metadata.interfaces';
     selector: 'adf-content-metadata',
     templateUrl: './content-metadata.component.html',
     styleUrls: ['./content-metadata.component.scss'],
-    host: { 'class': 'adf-content-metadata' },
+    host: { class: 'adf-content-metadata' },
     encapsulation: ViewEncapsulation.None
 })
 export class ContentMetadataComponent implements OnChanges, OnInit {
-
     /** (required) The node entity to fetch metadata about */
-    @Input()
-    node: MinimalNodeEntryEntity;
+    @Input() node: MinimalNodeEntryEntity;
```

Comment:
can you revert the indentation changes?

Reply:
I disabled Prettier to be able to autoformat again just by the tslint rules, but it looks like that we don't have any kind of rules for the indentation. Trying to autoformat again, nothing happens, looks like with Prettier we have stricter rules than with tslint (or put it simply, we just have rules for indentation)

Can it be, or I miss something?

Reply:
we could consider to apply as style the property and the decorates on the same line as suggested form the style guide of Angular but at the moment across all the project we have it on different lines, so for consistency let's revert this change.
About why this thing happen with the vs code extrension I guess this topic can help to understand why:
https://github.com/prettier/prettier/issues/1974